src/screens/FeedScreen/elements/Search.js

```diff
@@ -77,7 +77,7 @@ const styles = StyleSheet.create({
     // height: 36,
     paddingTop: 0,
     paddingBottom: 0,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the color of the text from `COLORS.blackgrey` to `COLORS.gray400` could impact readability, especially against certain backgrounds. Make sure the new color provides enough contrast against its background for optimal readability.
```diff
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->